CMakeLists.txt

```diff
@@ -788,6 +820,7 @@ configure_file(pkg-config/arpackSolver.pc.in "${PROJECT_BINARY_DIR}/arpackSolver
 
 
 install(TARGETS arpack
+    EXPORT arpackngTargets
```

Comment:
@juanjosegarciaripoll: could you just explain what you do here? not used to target export: why is this needed? (I try to improve my knowldge of cmake ! :))

Reply:
We instruct CMake to collect all the declarations we have made above (what include directories arpack depends on, what libraries it depends on, what link flags it needs to be built, etc) and create a file that exports that information about this particular target. A similar statement below applies to parpack as well. This information is the robust, relocatable way to export include and link flags from CMake projects. The developer does not need to write variables as it was done before.

Reply:
Thanks for clarifications!